src/api/api.controller.ts

```diff
@@ -150,4 +152,14 @@ export class ApiController {
     const apis = await this.apiService.getPopularAPis();
     return ZaLaResponse.Ok(apis, 'OK', '200');
   }
+
+  @Get('/contributors/:apiId')
+  @ApiOperation({summary: 'Get contributors of an API'})
+  async getApiContributors(
```

Comment:
Add IdCheck guard here to verify if apiId exists in the db,
Also call the Authorization and Authentication guard if needed.